build/nodejs_server/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM node:18.4.0
+
+ARG ENV
+ENV ENV=${ENV}
+
+WORKDIR /workspace
+
+# Install netstat which is useful for debugging.
+RUN apt-get -y update
+RUN apt-get -y install net-tools
+
+# Install dependencies
+COPY static/package.json /workspace/static/package.json
+COPY static/package-lock.json /workspace/static/package-lock.json
+WORKDIR /workspace/static
+RUN npm install --omit=dev
+
+COPY static/. /workspace/static/
+# Expose the port that the Node.js application will listen on
+EXPOSE 8080
```

Comment:
Add a comment that 8080 is crucial for health check as we see in google.com projects

Reply:
done